vuepress/docs/next/docs/consume/custom-resources.md

```diff
@@ -154,81 +149,16 @@ spec:
-   name previously registered with your DNS provider.
-* `spec.isDefault` is 'true' by default and this should suffice for most conditions. This will result in the standard 
-     `entando-cluster-infrastructure-secret` being replaced by a Secret connecting you to this newly created
-     Entando K8S Service.  Theoretically one could use multiple Entando K8S Services in a cluster, in which
-     case this property should be false for new Entando K8S Services that should not override the default Secret.
-* `spec.environmentVariables` is a Map of environment variables to pass to the Entando K8S Service Docker image. For example, this could
-     be used to override the ENTANDO_NAMESPACES_TO_OBSERVE variable that configures the set of Kubernetes namespaces that this
-     service should read the EntandoDeBundles from. Also note that all of the 
-     [Spring variables in entando-k8s-service project](https://github.com/entando-k8s/entando-k8s-service/blob/master/src/main/resources/application.properties)
-     can also be overridden here by specifying the equivalent SNAKE_CASE names of the dot-delimited Spring properties.
-     These parameters are applied to the container's environment variables after all variables have been calculated. 
-     It can therefore also be used as a mechanism to override any of the default environment variables that need customization.
-* `spec.tlsSecretName` is the name of a standard Kubernetes 
-     [TLS Secret](https://kubernetes.io/docs/concepts/services-networking/ingress/#tls) that will be used for the
-     resulting Ingress. This is only required if the 
-     [globally configured TLS Secret](https://github.com/entando-k8s/entando-k8s-controller-coordinator/blob/master/charts/entando-k8s-controller-coordinator/README.md#tls) 
-     for the Operator is absent or has not been created with a wildcard hostname that supports this Keycloak instance's hostname.
-* `spec.replicas` - the number of replicas to be made available on the Deployment of this Entando K8S Service.                                   
-* `spec.resourceRequirements` - the minimum and maximum [resource allocation](#the-resourcerequirements-specification) for the Entando Kubernetes Service container.                                     
+* `spec.resourceRequirements` - the minimum and maximum [resource allocations](#the-resourcerequirements-specification) for the Keycloak Server container.                                                               
 
 ## EntandoApp
 
 An EntandoApp is a Deployment of a Docker image that hosts an Entando and Java based web application. Entando offers two
 standard images that can be used, but generally we expect our customers to provide their images here. An EntandoApp
-Deployment packages three images into a single Pod: the Entando App Image in question, AppBuilder and Component Manager.
+deployment packages three images into a single Pod: the EntandoApp image discussed here, AppBuilder, and Component Manager.
```

Comment:
We'll have to fix this when we do the full rewrite. We don't have the 3 containers in 1 pod any more and we'd prefer users not customize the EntandoApp at all.